src/metabase/api/action.clj

```diff
@@ -23,7 +24,9 @@
   "Generic API endpoint for executing any sort of Action with source Table ID specified as part of the route."
   [action-namespace action-name table-id :as {:keys [body]}]
   (let [action (keyword action-namespace action-name)]
-    (actions/perform-action! action {:table-id table-id, :arg body})))
+    (actions/perform-action! action {:database (api/check-404 (table/table-id->database-id table-id))
```

Comment:
This was needed since `:database` is actually a required key in `:actions.args/common`. (I added this endpoint earlier today in #23922, which I spun off from this PR, but it was unused; this PR is the first thing to actually use it.)